jsonnet/thanos-ruler.libsonnet

```diff
@@ -1,457 +1,458 @@
 // TODO(paulfantom): This should use upstream kube-thanos project.
```

Comment:
should we remove this line?

Reply:
Absolutely. This is already done by pull request